cli/dcoscli/marathon/main.py

```diff
@@ -809,6 +829,48 @@ def pod_add(self, pod_resource_path):
         marathon_client.add_pod(pod_json)
         return 0
 
+    def pod_remove(self, pod_id, force):
+        """
+        :param pod_id: the Marathon ID of the pod to remove
+        :type pod_id: str
+        :param force: whether to override running deployments
+        :type force: bool
+        :returns: process return code
+        :rtype: int
+        """
+        return 0
+
+    def pod_list(self, json_):
+        """
+        :param json_: output JSON if true
+        :type json_: bool
+        :returns: process return code
+        :rtype: int
+        """
+        return 0
+
+    def pod_show(self, pod_id):
+        """
+        :param pod_id: the Marathon ID of the pod to remove
+        :type pod_id: str
+        :returns: process return code
+        :rtype: int
+        """
+        return 0
+
+    def pod_update(self, pod_id, properties, force):
+        """
+        :param pod_id: the Marathon ID of the pod to update
+        :type pod_id: str
+        :param properties: JSON items to update in pod definition
+        :type properties: [str]
+        :param force: whether to override running deployments
+        :type force: bool
+        :returns: process return code
+        :rtype: int
+        """
+        return 0
```

Comment:
lets change the return values of these methods to `throw DCOSException("Not implemented") so it's more explicit